orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf

```diff
@@ -24,28 +24,30 @@ module "orc8r" {
 
   # If you performing a fresh Orc8r install, choose a recent Postgres version
   # orc8r_db_engine_version     = "12.6"
-  orc8r_db_password = "mypassword" # must be at least 8 characters
+  orc8r_db_password = "password" # must be at least 8 characters
 
   setup_cert_manager = false
 
   secretsmanager_orc8r_secret = "orc8r-secrets"
-  orc8r_domain_name           = "orc8r.example.com"
+  orc8r_domain_name           = "orc8r.magmacore.link"
 
-  orc8r_sns_email             = "admin@example.com"
+  orc8r_sns_email             = "admin@magmacore.link"
   enable_aws_db_notifications = true
 
   vpc_name     = "orc8r"
   cluster_name = "orc8r"
 
   deploy_elasticsearch          = true
-  elasticsearch_domain_name     = "orc8r-es"
-  elasticsearch_version         = "7.7"
-  elasticsearch_instance_type   = "t2.medium.elasticsearch"
-  elasticsearch_instance_count  = 2
-  elasticsearch_az_count        = 2
-  elasticsearch_ebs_enabled     = true
-  elasticsearch_ebs_volume_size = 32
-  elasticsearch_ebs_volume_type = "gp2"
+  # elasticsearch_domain_name     = "orc8r-es"
+  # elasticsearch_version         = "7.7"
+  # elasticsearch_instance_type   = "t2.medium.elasticsearch"
+  # elasticsearch_instance_count  = 2
+  # elasticsearch_az_count        = 2
+  # elasticsearch_ebs_enabled     = true
+  # elasticsearch_ebs_volume_size = 32
+  # elasticsearch_ebs_volume_type = "gp2"
+
+  deploy_elasticsearch_service_linked_role = false
```

Comment:
We only make it `false` if we are deploying multiple Orc8r on the same AWS account. For the first time deployment this should be `true` which is the default value.